src/python_testing/TC_EEVSE_2_2.py

```diff
@@ -0,0 +1,319 @@
+            TestStep("15", "TH sends command Disable. Verify Event EEVSE.S.E03(EnergyTransferStopped) sent with reason EvseStopped"),
+            TestStep("15a", "TH reads from the DUT the SupplyState attribute. Verify value is 0x00 (Disabled)"),
+            TestStep("16", "TH sends TestEventTrigger command to General Diagnostics Cluster on Endpoint 0 with EnableKey field set to PIXIT.EEVSE.TEST_EVENT_TRIGGER_KEY and EventTrigger field set to PIXIT.EEVSE.TEST_EVENT_TRIGGER for EV Charge Demand Test Event Clear."),
+            TestStep("17", "TH sends TestEventTrigger command to General Diagnostics Cluster on Endpoint 0 with EnableKey field set to PIXIT.EEVSE.TEST_EVENT_TRIGGER_KEY and EventTrigger field set to PIXIT.EEVSE.TEST_EVENT_TRIGGER for EV Plugged-in Test Event Clear. Verify Event EEVSE.S.E01(EVNotDetected) sent"),
+            TestStep("18", "TH sends TestEventTrigger command to General Diagnostics Cluster on Endpoint 0 with EnableKey field set to PIXIT.EEVSE.TEST_EVENT_TRIGGER_KEY and EventTrigger field set to PIXIT.EEVSE.TEST_EVENT_TRIGGER for Basic Functionality Test Event Clear."),
+        ]
+
+        return steps
+
+    @async_test_body
+    async def test_TC_EEVSE_2_2(self):
+
+        self.step("1")
+        # Commission DUT - already done
+
+        # Subscribe to Events and when they are sent push them to a queue for checking later
+        events_callback = EventChangeCallback(Clusters.EnergyEvse)
+        await events_callback.start(self.default_controller,
+                                    self.dut_node_id,
+                                    self.matter_test_config.endpoint)
+
+        self.step("2")
+        await self.check_test_event_triggers_enabled()
+
+        self.step("3")
+        await self.send_test_event_trigger_basic()
+
+        # After a few seconds...
+        time.sleep(1)
+
+        self.step("3a")
+        await self.check_evse_attribute("State", Clusters.EnergyEvse.Enums.StateEnum.kNotPluggedIn)
+
+        self.step("3b")
+        await self.check_evse_attribute("SupplyState", Clusters.EnergyEvse.Enums.SupplyStateEnum.kDisabled)
+
+        self.step("3c")
+        await self.check_evse_attribute("FaultState", Clusters.EnergyEvse.Enums.FaultStateEnum.kNoError)
+
+        self.step("4")
+        await self.send_test_event_trigger_pluggedin()
+        event_data = events_callback.WaitForEventReport(Clusters.EnergyEvse.Events.EVConnected)
+
+        self.step("4a")
+        await self.check_evse_attribute("State", Clusters.EnergyEvse.Enums.StateEnum.kPluggedInNoDemand)
+
+        self.step("4b")
+        # Save Session ID for later and check it against the value in the event
+        session_id = await self.read_evse_attribute_expect_success(attribute="SessionID")
+        self.validate_ev_connected_event(event_data, session_id)
+
+        self.step("5")
+        charging_duration = 5  # TODO test plan spec says 120s - reduced for now
```

Comment:
Is this still a reasonable test? ie, is the intent here to change the test plan, or was this reduced for development and not changed back?

Note that we do have a PICS for CI so if you want to differentiate the CI from the cert test, that's also possible.

Reply:
I think we should probably change the test plan to a smaller number too, originally this 2 mins was chosen because the implementation around timers wasn't certain (e.g. the EVSE might run a 1 min timer check, but now it is implemented it can be a much smaller number). 

Its not currently in the CI checkers either - I'll raise a follow up on both these aspects and to change this value to a suitable number.

Reply:
Added Follow Issue https://github.com/project-chip/connectedhomeip/issues/31460